src/app/language-support/page.tsx

```diff
@@ -0,0 +1,67 @@
+'use client';
+
+import { getAllLanguageSupport } from '@/api/supabase/queries/language-support';
+import { useEffect, useState } from 'react';
+import { LanguageSupport } from '@/types/schema';
+
+export default function Page() {
+  const [languageSupport, setLanguageSupport] = useState<LanguageSupport[]>([]);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const data = await getAllLanguageSupport();
+        setLanguageSupport(data as LanguageSupport[]);
```

Comment:
do we need the `as LanguageSupport[]` here? I would assume that Typescript already infers it due to the return type of `getAllLanguageSupport`

Reply:
gotchu, changed!